river/tree/_attribute_observer/nominal_attribute_class_observer.py

```diff
@@ -19,6 +19,10 @@ def __init__(self):
         self._missing_weight_observed = 0.0
         self._att_val_dist_per_class = {}
 
+    @property
+    def is_numeric(self):
```

Comment:
Nice :-)

Reply:
We might consider dropping `Class` and `Regression/Reg` from the AO/Splitter names and using a property instead.